tests/store/test_status_cases_filters.py

```diff
@@ -274,3 +276,54 @@ def test_filter_cases_for_analysis_when_cases_with_no_action_and_old_sequence_da
 
     # THEN cases should not contain the test case
     assert not cases
+
+
+def test_filter_cases_with_scout_data_delivery(
+    base_store: Store, helpers: StoreHelpers, timestamp_today: datetime
+):
+    """Test that a case is returned when Scout is specified as a data delivery option"""
```

Comment:
```suggestion
    """Test that a case is returned when Scout is specified as a data delivery option."""
```
I decided to actually look up what python thinks is required in a docstring and apparently it is 2 things: Capital letter in the beginning of the docstring and a dot at the end (i have previously always removed the dot). Since, it should be there I think we should start adding it again. 😓

Reply:
Nice 👍